docs/source/array-random.rst

```diff
@@ -1,5 +1,5 @@
-dask.array.random
-=================
+Random Arrays
+=============
 
 ``dask.array`` copies the ``numpy.random`` module for all univariate
 distributions.  The interface to each function is identical except for the
```

Comment:
This is nice. I did the same for the `bag` and `dataframe` API pages. I removed the `array-random` subpage references in this thread in lieu of the updated API pages.